app/scripts/lib/router.js

```diff
@@ -103,6 +104,7 @@ define(function (require, exports, module) {
       'settings/display_name(/)': createChildViewHandler(DisplayNameView, SettingsView),
       'settings/emails(/)': createChildViewHandler(EmailsView, SettingsView),
       'signin(/)': createViewHandler(SignInView),
+      'signin_bounced(/)': createViewHandler(SignInBouncedView),
```

Comment:
Can you entries for this endpoint in both [tests/functional/pages.js](https://github.com/mozilla/fxa-content-server/blob/f9f7f10d4310163ffc256d1536322d470dcfe646/tests/functional/pages.js#L60) and [tests/server/routes.js](https://github.com/mozilla/fxa-content-server/blob/f9f7f10d4310163ffc256d1536322d470dcfe646/tests/server/routes.js#L77)?